charts/thanos/README.md

```diff
@@ -377,5 +351,4 @@ helm upgrade --install thanos stevehipwell/thanos --version 1.20.0
 | storeGateway.updateStrategy | object | `{}` | Update strategy for the _Store Gateway_ stateful set. |
 
 ----------------------------------------------
-
-Autogenerated from chart metadata using [helm-docs](https://github.com/norwoodj/helm-docs/).
+Autogenerated from chart metadata using [helm-docs v1.14.2](https://github.com/norwoodj/helm-docs/releases/v1.14.2)
```

Comment:
You need to run `helm-docs` with the custom args as described in the contributing docs.